servicemanager/smprocess.py

```diff
@@ -176,7 +172,7 @@ def __init__(self, ppid, pid, uptime, mem, args):
     def processes_matching(regex, processes=None):
         processes = processes or SmProcess.all_processes()
```

Comment:
I hope it is - see https://github.com/hmrc/service-manager/blob/8ccfd2134bf254f9eeda01ef9ec38c12011c6382/servicemanager/smstatus.py#L55